services/cd-service/pkg/cmd/server.go

```diff
@@ -91,6 +91,8 @@ func RunServer() {
 			defer tracer.Stop()
 		}
 
+		span, _ := tracer.StartSpanFromContext(ctx, "Start server")
```

Comment:
If you discard the ctx return value, then your span will be propagated through the stack. Please use
```
span, ctx := ...
```